components/_share.scss

```diff
@@ -12,17 +9,24 @@
 
 .sharing-box {
   clear: both;
-  border: 2px solid #F0F0F1;
+  border: $base-border;
+  border-radius: $base-border-radius;
   margin: auto;
   padding: 20px;
-  background-color: #FFFFFF;
+  background-color: $tws-white;
+
+  @include media($tablet) {
+    @include span-columns(10 of 12);
+    @include shift(1);
+  }
 
   @include media($desktop) {
-    width: 465px;
+    @include span-columns(8 of 12);
+    @include shift(2);
   }
 
   .sharing-preview {
-    border: 1px solid #FCE0D7;
+    border: 1px solid $tws-yellow;
```

Comment:
Properties should be ordered -webkit-animation-duration, -webkit-animation-duration, animation-duration, animation-duration, -webkit-animation-fill-mode, animation-fill-mode, border, border-radius